olp-cpp-sdk-dataservice-write/include/olp/dataservice/write/StreamLayerClient.h

```diff
@@ -79,6 +79,20 @@ class DATASERVICE_WRITE_API StreamLayerClient {
                     StreamLayerClientSettings client_settings,
                     client::OlpClientSettings settings);
 
+  /**
+   * @brief Cancels all pending operations started by this client.
+   *
+   * Cancels all on-going publish operations, which are started by this client.
+   * It returns instantly without waiting for the callbacks on these
```

Comment:
```suggestion
   * It returns instantly and does not wait for the callbacks.
```
Just simplified it a bit